contrib/inventory/vmware_inventory.py

```diff
@@ -418,7 +418,9 @@ def _get_instances(self, inkwargs):
                         self.custom_fields[f.key] = f.name
                 self.debugl('%d custom fields collected' % len(self.custom_fields))
         except vmodl.RuntimeFault as exc:
-            self.debugl("Unable to gather custom fields due to %s" % exc.msg)
+            self.debugl("Unable to gather custom fields due to %s" % exc)
```

Comment:
Small nit pick here - Do not remove `msg` attribute from `exc`, it is helpful in debugging. Rest LGTM

Reply:
All Set thanks!